metabase/metabase
# Allow selecting only one card for a parameter values source in a dashboard

Epic https://github.com/metabase/metabase/issues/26898

How to verify:
- Create a dashboard with a card and a `text -> dropdown` parameter mapped to a column in the card
- Edit dashboard -> Select parameter -> Click `Values from a model or question`
- Make sure it is possible to select only one question or model

<img width="1140" alt="Screenshot 2023-01-04 at 17 19 56" src="https://user-images.githubusercontent.com/8542534/210588199-9c12bc0e-9f13-4dc8-9c43-8c80d1ceede6.png">
